site/usage/applications.md

```diff
@@ -35,9 +36,15 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 -   [Vegas](https://github.com/aishfenton/Vegas) brings visualizations to Scala and Spark using Vega-Lite.
 -   [Vizard](https://github.com/yieldbot/vizard) tiny REPL client to visualize Clojure data in browser w/ Vega-Lite.
 -   [oz](https://github.com/metasoarous/oz) Vega & Vega-Lite based visualization toolkit for Clojure & ClojureScript (fork of Vizard).
+-   [Saite](https://github.com/jsa-aerial/saite) Exploratory graphics and ad hoc visualization system. Built in Clojure(Script) on top of [Hanami](https://github.com/jsa-aerial/hanami).
```

Comment:
I might misunderstand, but I think of as a more high level tool so it should go under "Tools that use Vega-Lite". We put low level libraries in this group.

Reply:
Well, it is unclear to me. Altair is certainly rather higher level than say Oz (and so Vizard, which is really almost nothing), but both of those are also 'things' that make visualizations as opposed to 'bindings'.

Hmmmm, the more I think about it, Hanami may go better under this 'Bindings ... ' heading as it is really at a level which 1.) you create VG/VGL specs using higher level parameterized templates (which are transformed to finished specs) coupled with various capabilities for creating client only or client/server applications based on the templates.

Saite is actually a complete mini client/server application built on top of these Hanami capabilities so it probabably would fit better under "Tools that use..."

What do you think of that placement??

Reply:
I trust you to fit it under existing categories.